data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/validator/GetNewBlindedBlock.java

```diff
@@ -167,19 +173,4 @@ private static EndpointMetadata getEndpointMetaData(
                 block -> spec.getForkSchedule().getSpecMilestoneAtSlot(block.getSlot())))
         .build();
   }
-
-  private static SerializableOneOfTypeDefinition<BeaconBlock> getBlindedBlockSchemaDefinition(
```

Comment:
This cleanup basically allows us to filter the capella milestone when listing types of blinded blocks... doing our own handling here was going to mean a lot of duplication of filters.